Essentials/src/com/earth2me/essentials/commands/EssentialsLoopCommand.java

```diff
@@ -129,4 +139,8 @@ protected void loopOnlinePlayers(final Server server, final CommandSource sender
         players.add("*");
         return players;
     }
+
+    public interface UserConsumer {
```

Comment:
Is this interface really necessary? Why not `Consumer<User>`?

Reply:
Consumers are not allowed to throw exceptions